Cargo.toml

```diff
@@ -25,6 +25,21 @@ cast_precision_loss = "deny"
 ptr_as_ptr = "deny"
 useless_conversion = "deny"
 
+[workspace.dependencies]
+mongodb = { version = "3", features = ["aws-auth", "dns-resolver"] }
```

Comment:
In general let's be suspicious of including features in the workspace dependencies, as then sub-crates will have to take in those features when they compile, even if they don't use them. In this case and for `serde` down below I think it's fine as any future use we'll put on the author to change it up here if needed.